corehq/blobs/tests/test_init.py

```diff
@@ -29,6 +29,7 @@ def test_get_blobdb(self, msg, root=True, blob_dir=None):
             temp_dir=None,
             blob_dir=blob_dir,
         )
+        mod._db = []  # reset singleton
```

Comment:
This is necessary to ensure that we aren't fetching a blobdb from the stack since we're testing instantiation of the DB. Updated to use a patch here: 2de336a6c4873cd2c1d5bc9ebfaa92b7a53b2841